powerfactory_utils/exporter/exporter.py

```diff
@@ -1332,8 +1331,8 @@ def create_q_controller(
             elif controller_type == ControllerType.Q_CONST:
                 q_set = gen.qgini
             elif controller_type == ControllerType.Q_U:
-                qmax_ue = abs(gen.Qfu_min / p_r)  # p.u.
-                qmax_oe = abs(gen.Qfu_max / p_r)  # p.u.
+                qmax_ue = abs(gen.Qfu_min)
+                qmax_oe = abs(gen.Qfu_max)
```

Comment:
```suggestion
                qmax_oe = round(abs(gen.Qfu_max), DecimalDigits.POWER)
```

Reply:
decline. rounding is intended at the end of the function, right?
q_r is oinly used for qmax and q_max is rounded at the end.